packages/astro/e2e/view-transitions.test.js

```diff
@@ -631,7 +631,7 @@ test.describe('View Transitions', () => {
 	});
 
 	test('client:only styles are retained on transition', async ({ page, astro }) => {
-		const totalExpectedStyles = 8;
+		const totalExpectedStyles = 7;
```

Comment:
There's no more `<style data-astro-dev-id` as it's merged into `<style data-vite-dev-id`, so this number is reduced by one.